<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>782</width>
    <height>588</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <family>Courier New</family>
   </font>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <item row="2" column="0" colspan="2">
    <widget class="QTextBrowser" name="PfOut">
     <property name="toolTip">
      <string>Output as dict; base as key and exponent as value</string>
     </property>
     <property name="placeholderText">
      <string>Output as dict</string>
     </property>
    </widget>
   </item>
   <item row="0" column="0">
    <widget class="QLabel" name="label">
     <property name="maximumSize">
      <size>
       <width>45</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="toolTip">
      <string>Number to prime factorize</string>
     </property>
     <property name="text">
      <string>Number</string>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QSpinBox" name="PfInput">
     <property name="toolTip">
      <string>Number to prime factorize</string>
     </property>
     <property name="minimum">
      <number>2</number>
     </property>
     <property name="maximum">
      <number>999999999</number>
     </property>
    </widget>
   </item>
   <item row="3" column="0" colspan="2">
    <widget class="QTextBrowser" name="PfApprox">
     <property name="toolTip">
      <string>Output as string; (a**b)</string>
     </property>
     <property name="placeholderText">
      <string>Output as string</string>
     </property>
    </widget>
   </item>
   <item row="1" column="0" colspan="2">
    <widget class="QPushButton" name="PfCalc">
     <property name="toolTip">
      <string>Factorize number</string>
     </property>
     <property name="text">
      <string>Calculate</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
